CHANGELOG.md

```diff
@@ -100,6 +100,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - [Segment Replication] Add point in time and scroll query compatibility. ([#6644](https://github.com/opensearch-project/OpenSearch/pull/6644))
 - Add retry delay as dynamic setting for cluster maanger throttling. ([#6998](https://github.com/opensearch-project/OpenSearch/pull/6998))
 - Introduce full support for searchable snapshots ([#5087](https://github.com/opensearch-project/OpenSearch/issues/5087))
+- Introduce Segment Replication as General Availability Feature ([#5147](https://github.com/opensearch-project/OpenSearch/issues/5147))
```

Comment:
Nitpick, I'd phase this the same as above. Either phrase both lines as "Introduce <FEATURE> as a generally available feature" or as "Introduce full support for <FEATURE>". Either way is fine with me.